notebooks/py/assay_comparison.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+# coding: utf-8
+#
+# EPY: stripped_notebook: {"metadata": {"kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.6.5"}}, "nbformat": 4, "nbformat_minor": 2}
+
+# EPY: START code
+# plotting
+# EPY: ESCAPE %matplotlib inline
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+sns.set_context('talk')
+sns.set_style('ticks')
+# EPY: END code
+
+# EPY: START code
+# munging
+import os
+# EPY: END code
+
+# EPY: START code
+# science
+from starfish import IntensityTable, Experiment, ImageStack
+from starfish.plot.intensity_table import compare_copy_number, spot_area_histogram, spot_distance_histogram, barcode_magnitude_histogram
+from starfish.plot import spots
+from starfish.types import Indices
+import numpy as np
+import pandas as pd
+
+from showit import image
+# EPY: END code
+
+# EPY: START markdown
+### Load IntensityTables
+# EPY: END markdown
+
+# EPY: START code
+# IntensityTable can't download from directories without list privileges
+
+data_root = "https://dmf0bdeheu4zf.cloudfront.net/assay_comparison/"
+iss_link = data_root + "iss.nc"
```

Comment:
@ttung and marcus always make me do os.path.join(dir, stuff, stuff2, ... stuffN). i noticed that that's way better than using + as it handles slashes nicely